cmake/ArangoDBInstall.cmake

```diff
@@ -81,11 +81,40 @@ install(
     ${PROJECT_SOURCE_DIR}/js/apps
     ${PROJECT_SOURCE_DIR}/js/server
   DESTINATION ${CMAKE_INSTALL_DATAROOTDIR_ARANGO}/${ARANGODB_JS_VERSION}
-  REGEX       "^.*/aardvark/APP/react/node_modules$"       EXCLUDE
+  REGEX       "^.*/aardvark/APP/frontend.*$"               EXCLUDE
+  REGEX       "^.*/aardvark/APP/react.*$"                  EXCLUDE
   REGEX       "^.*/js/server/assets/swagger/*.map$"        EXCLUDE
   REGEX       "^.*/.bin"                                   EXCLUDE
 )
 
+set(APP_FILES
+ "frontend/img"
+ "react/build"
+ "react/public/assets/aqltemplates.json"
```

Comment:
```suggestion

```

Reply:
Can be removed. I changed the manifest in structure in another PR. So that file is now also available in build automatically. To keep things as simple as we can here.